patches/0004-Output-to-hashed-mappings.patch

```diff
@@ -0,0 +1,1622 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: tibs <git@tibinonest.me>
+Date: Wed, 11 May 2022 15:52:47 -0400
+Subject: [PATCH] Output to hashed mappings
+
+Includes merging intermediary and hashed into one main mappings tree so that dependencies can be remapped when required.
+
+diff --git a/src/main/java/net/fabricmc/loom/LoomGradleExtension.java b/src/main/java/net/fabricmc/loom/LoomGradleExtension.java
+index c047822dedfcf9833ddf1ab73f9b1e4cb0722972..78f88b0143e20e69bc1ab27bbe6f370b10e9ff82 100644
+--- a/src/main/java/net/fabricmc/loom/LoomGradleExtension.java
++++ b/src/main/java/net/fabricmc/loom/LoomGradleExtension.java
+@@ -47,6 +47,7 @@ import net.fabricmc.loom.configuration.accesswidener.AccessWidenerFile;
+ import net.fabricmc.loom.configuration.processors.JarProcessorManager;
+ import net.fabricmc.loom.configuration.providers.mappings.MappingsProviderImpl;
+ import net.fabricmc.loom.configuration.providers.minecraft.MinecraftProvider;
++import net.fabricmc.loom.configuration.providers.minecraft.mapped.HashedMinecraftProvider;
+ import net.fabricmc.loom.configuration.providers.minecraft.mapped.IntermediaryMinecraftProvider;
+ import net.fabricmc.loom.configuration.providers.minecraft.mapped.NamedMinecraftProvider;
+ import net.fabricmc.loom.extension.LoomFiles;
+@@ -99,14 +100,19 @@ public interface LoomGradleExtension extends LoomGradleExtensionAPI {
+ 
+ 	IntermediaryMinecraftProvider<?> getIntermediaryMinecraftProvider();
+ 
++	HashedMinecraftProvider<?> getHashedMinecraftProvider();
++
+ 	void setNamedMinecraftProvider(NamedMinecraftProvider<?> namedMinecraftProvider);
+ 
+ 	void setIntermediaryMinecraftProvider(IntermediaryMinecraftProvider<?> intermediaryMinecraftProvider);
+ 
++	void setHashedMinecraftProvider(HashedMinecraftProvider<?> hashedinecraftProvider);
```

Comment:
There's a typo of `Minecraft` in the parameter name

Reply:
Thx lol